integrated_channels/integrated_channel/transmitters/learner_data.py

```diff
@@ -219,6 +220,9 @@ def transmit(self, payload, **kwargs):
                     getattr(learner_data, kwargs.get('remote_user_id')),
                     serialized_payload
                 )
+                if code >= HTTPStatus.BAD_REQUEST.value:
```

Comment:
torn on if we need that message or we can just use `body` here

Reply:
Given that we've seen mixed response types coming from the differing LMS', I feel like it would be worth to have some constant identifying chars with the log.

Reply:
I see, so this catches anytime where a `400+` slips through any of the channels without us raising a client error. Okay, I like this approach 👍